ONSdigital/design-system
# Include domready in package

### What is the context of this PR?
- Aims to include the 'domready.js' file in published npm packages (see #1761).
- Fix issues with file handling and minor cleanup.

### How to review
- Run `yarn npm-bundle` and `yarn cdn-bundle` locally to verify output.
- Create a test version on npm to verify that 'domready.js' is now included.
